tests/smoke/install/task.yaml

```diff
@@ -44,9 +44,9 @@ execute: |
     snap changes | MATCH 'Install "test-snapd-sh" snap'
 
     echo "Ensure different bases work"
-    for base in core18 core20; do
-        # no core20 snap for i386
-        if [ "$base" = "core20" ] && [ "$(uname -m)" = i686 ]; then
+    for base in core18 core20 core22; do
+        # no core20 or core22 snap for i386
+        if { [ "$base" = "core20" ] || [ "$base" = "core22" ]; } && [ "$(uname -m)" = i686 ]; then
```

Comment:
Hm, silly question, this `if {` pattern I only see rarely, usually I see `(` type brackets, why the `{` here? Is there an advantage?

Reply:
Good question! I was hit over the head by shellcheck telling me this is the correct way to do this.

Reply:
Haha - nice. shellcheck is the best. It's fine if it suggested it. It's just a bit usual I gues